pvlib/data/adr-library.csv

```diff
@@ -0,0 +1,1762 @@
+Name,Manufacturer,Model,Source,Vac,Vintage,Pacmax,Pnom,Vnom,Vmin,Vmax,ADRCoefficients,Pnt,Vdcmax,Idcmax,MPPTLow,MPPTHi,TambLow,TambHi,Weight
+0,0,0,0,0.0,0.0,0.0,0,0,0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
```

Comment:
I'm struggling to think of a good solution for this and the array parsing issue. One relatively simple option is to replace the 0 lines with metadata lines like in the sam files. So, we'd want a units line and then I guess a copy of the column names. Or maybe just empty cells instead of 0s would be better? The end of the parser function could have a conditional on the presence of 'ADRCoefficients' for the array parsing issue discussed above.